packages/mds-event-processor/src/proc.ts

```diff
@@ -0,0 +1,81 @@
+import db from '@mds-core/mds-db'
+import cache from '@mds-core/mds-cache'
+import http from 'http'
+import log from '@mds-core/mds-logger'
+
+const env = process.env
+
+async function resetAll(type: string) {
```

Comment:
This function does some dangerous stuff.

Should we do some env var checks to make sure this doesn't get invoked on prod accidentally?